lib/Settings/Admin.php

```diff
@@ -58,7 +58,7 @@ public function __construct(IConfig $config,
         ) {
                 $this->config = $config;
                 $this->l = $l;
-		$this->initialStateService = $initialStateService;
+				$this->initialStateService = $initialStateService;
```

Comment:
Seems off-place; inspecting further, the other lines are padded with 8 spaces, this one was with 2 tabs and the PR would add more tabs.
```suggestion
                $this->initialStateService = $initialStateService;
```

Reply:
Right, looks like this was caused by Nextcloud's [.editorconfig](https://github.com/nextcloud/server/blob/master/.editorconfig).
I fixed it in the latest version

Reply:
So just that I understand this right: Your desired indentation format is 8 spaces, right?

Reply:
Ah, actually, my bad. Seems that the file has mixed tabs and spaces indentation. Feel free to ignore my earlier comment, I'll clean up the whole file later.

Reply:
@Mazhoon I saw that you've been using mostly tabs before - I tried to cleanup my commits and adjust the code style :)
I hope, everything is alright now